src/components/publishing/image_collection_item.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react"
+import styled from "styled-components"
+
+interface FillwidthItemProps extends React.HTMLProps<HTMLDivElement> {
+  width?: any
```

Comment:
Hahaha. To be fair, we did tell @kanaabe and @eessex that they don’t need to add typings immediately.